integration-testing/elasticurl_test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Is it worth writing this in Python so we don't need a separate script for Windows?